src/main/java/ch/naviqore/service/Walk.java

```diff
@@ -0,0 +1,18 @@
+package ch.naviqore.service;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public interface Walk {
+    @NotNull Location getStart();
```

Comment:
Redundant, since can be retrieved via Stop. Except we support walks from random locations to stops. E.g. from an address to the station. But I would keep it simple and extend the interface if we need it.